webcomponents/tool/src/main/frontend/packages/sakai-tasks/test/sakai-tasks-create-task.test.js

```diff
@@ -10,7 +10,7 @@ import fetchMock from "fetch-mock/esm/client";
 describe("sakai-tasks-create-task tests", () => {
 
   const minusFiveHours = -5 * 60 * 60 * 1000;
-  window.top.portal = { locale: "en_GB", user: { offsetFromServerMillis: minusFiveHours } };
+  window.top.portal = { locale: "en_GB", user: { offsetFromServerMillis: minusFiveHours, timezone: "America/New_York" } };
```

Comment:
Maybe a nitpick and not really changed by of this but _seems_ like with the explicit `America/New_York` timezone the locale should be `en_US` instead of `en_GB`. Same of the other test below.